crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -49,6 +49,7 @@ fn validate_version_in_params(
             version
                 .as_str()
```

Comment:
The reason for failure can be the use of `as_str` function returns `Some` value only when the JSON value is a `String`.

This can be resolved by doing `version.as_i64`

Reply:
yeah we want to support the user to send the data in both the formats, thats why didn't replace the previous set of lines with this

Reply:
"" are unnecessary in the query param value